letsencrypt/tests/cli_test.py

```diff
@@ -93,6 +94,15 @@ def test_help(self):
             from letsencrypt import cli
             self.assertTrue(cli.USAGE in out)
 
+    def test_configurator_selection(self):
+        plugins = disco.PluginsRegistry.find_all()
```

Comment:
Please don't rely on `letsencrypt-apache` being installed, but mock out results of `PluginRegistry.find_all`